webapp/src/services/near-nep21-util.js

```diff
@@ -100,16 +106,16 @@ export async function swapFromIn( token1, token2 ) {
       const price = await window.contract.swap_tokens_exact_in( {
         from: token1.address,
         to: token2.address,
-        tokens_from: 2, // ??
-        min_tokens_to: 2 }); // ??
+        tokens_from: amount1, 
+        min_tokens_to: amount2 }); 
       return price;
     }
     else {
```

Comment:
We must have a check in this `else` that `token2.type == native`, and add else to **log error** otherwise.